### DISCUSSION TITLE
Current Model in-use (selected) at the top

### DISCUSSION THREAD

**Discussion by arimxyer** [cline/cline#730] [Feature Requests]
Would be helpful to see the current model selected or in use at the top - so that in the event that you switch, or are unaware of what model you're currently working with, you can quickly see it at the top.

![image](https://github.com/user-attachments/assets/28a1f388-2799-40b8-b4dd-db61bae9f58e)

**Comment by dcbartlett** [comment]
We added this to the bottom of the chat just under the prompt box.  Does this work?
